docs/ref/templates/builtins.txt

```diff
@@ -711,6 +711,12 @@ available to the included template::
     been evaluated and rendered* - not blocks that can be overridden by, for
     example, an extending template.
 
+.. note::
+    If the included template is missing, the effect varies depending on the
+    :setting:`DEBUG` setting: when ``DEBUG`` mode is on, a
```

Comment:
It's actually dependent on `engine.debug` ('debug' key of [DjangoTemplates](https://docs.djangoproject.com/en/1.8/topics/templates/#module-django.template.backends.django) engine).